docs/React.README.md

```diff
@@ -1,146 +0,0 @@
-# Front-end Developer Guide to LORIS
-
->**Note**: This guide assumes basic knowledge of **Javascript** and **React JS**
-
-## Overview
-
-Currently Loris is in the process of "reactification" of its existing modules. This implies that:
-- Major module updates should rely on React for all the frontend needs `(#GetRidOfSmarty)`
-- New modules should be built using Loris React components (both existing and new)
-
----
-
-## Coding Guidelines
-- All new React components and major updates to existing React components should follow `ES6 standard`
-- All new and existing Javascript code should pass `ESLint` validation
-- All new components should be compiled using `webpack`
-- All new modules and major module updates should use a single javascript entry-point `index.js` and rely on ES6 modules to include additional components
-
----
-
-## File Structure
-
-**1. Generic `JSX` files (Loris Core)**
-
-Within LORIS we have many React components that can be used throughout the different modules. When adding a new generic file, the `JSX` file should be placed in the `jsx/` directory and its compiled file in the `htdocs/js/components` directory.
-
-**2. Module specific JSX files**
-
-In many cases, React components are designed for a specific LORIS module. In such cases the `JSX` file should be placed in the
-`module/$DESIRED_MODULE/jsx/` and its compiled file in the `module/$DESIRED_MODULE/js/` directory.
-
-
->**Note**: When creating a new JSX file **always** use the `.js` extension or the continuous compilation will not work as expected. The file name of the `JSX` and its compiled file should have the same name.
-
----
-
-## Set up
```

Comment:
@driusan i still use this section everytime I have to setup a new VM or instruct someone to install LORIS. If we can move this the some sort of FAQ somewhere else I'll approve the changes

Reply:
Which part is missing? The CentOS part is already in `docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md` and the Ubuntu part should be in the Ubuntu equivalent, but I haven't verified

Reply:
(Also the instructions here are wrong and install an outdated version, at least compared to the RTD documentation version..)

Reply:
you can merge it then I guess we'll revise the install instructions if people complain

Reply:
If the install instructions are missing something we should be updating the install instructions anyways.. people (other than @ridz1208, apparently..) wouldn't think to look into `React.README.md` for LORIS install dependencies.

Reply:
I only look in there cause Alex Ilea sent that file to me the first time lol.

However if we dont instruct users on how to install dependencies like git or php... maybe we dont need it for node and NPM either